src/styles/styles.js

```diff
@@ -37,6 +37,10 @@ const styles = {
         color: themeColors.linkHover,
     },
 
+    cWhite: {
+        color: themeColors.textReversed,
+    },
```

Comment:
this is redundant as we already have `colorReversed`

https://github.com/Expensify/Expensify.cash/blob/c1536a83ef628f27dcae5d0320bfba16d7bff58e/src/styles/styles.js#L109-L111